source/utilities.tex

```diff
@@ -687,7 +687,7 @@
 \expects
 \tcode{false} is \tcode{true}.
 \begin{note}
-This precondition cannot be satisfied, thus the behavior
+Since this precondition cannot be satisfied, the behavior
```

Comment:
Richard was avoiding "Since", because it might be interpreted temporal "since two weeks ago".  Use "because" or "thus".

Reply:
That's a good point, I'll add a fixup.

Reply:
That said, we can avoid "cannot": "The precondition is never satisfied, thus..."